### DISCUSSION TITLE
Hello, World!

### DISCUSSION THREAD

**Discussion by XenHat** [Open-Shell/Open-Shell-Menu#577] [General]
Well, this is a thing now.
I'm curious to see how this feature works out for us :smile_cat:

**Comment by XenHat** [comment]
No notifications is going to be a real pain in the butt, Rather use Gitter or discord still.

**Reply by bonzibudd** [reply]
Notifications seem to be working for me, at least for email.

**Reply by ge0rdi** [reply]
Works for me too.

**Comment by XenHat** [comment]
oh hey, mine works now!

**Comment by coddec** [comment]
Interesting, doesn't seem to be polished yet, well it's still in Beta, but hey, this is a thing now.
Maybe one day, we can move from Gitter back to here?
Though, Gitter has their own App for that purpose, I know GitHub has App too.
Also, Microsoft is famous or known to be removing whatever people thinks useful or great..... So.... maybe let's see how this feature goes first?
